mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/ResourceClusterScalerActor.java

```diff
@@ -239,9 +239,43 @@ private void onGetClusterIdleInstancesResponse(GetClusterIdleInstancesResponse r
 
     private void onTriggerClusterUsageRequest(TriggerClusterUsageRequest req) {
         log.trace("Requesting cluster usage: {}", this.clusterId);
+        if (this.skuToRuleMap.isEmpty()) {
+            log.info("{} scaler is disabled due to no rules", this.clusterId);
+            return;
+        }
         this.resourceClusterActor.tell(new GetClusterUsageRequest(this.clusterId), self());
     }
 
+    private void onTriggerClusterRuleRefreshRequest(TriggerClusterRuleRefreshRequest req) {
+        log.info("{}: Requesting cluster rule refresh", this.clusterId);
+        this.fetchRuleSet();
+    }
+
+    private void fetchRuleSet() {
+        CompletionStage<GetRuleSetResponse> fetchFut =
+            this.storageProvider.getResourceClusterScaleRules(this.clusterId.getResourceID())
```

Comment:
nit: send in the cluster ID to the getResourceClusterScaleRules directly. Let it handle the serialization/deserialization logic from the underlying storage. Dealing with stronger types is always better, IMO.